streams/src/test/java/org/apache/kafka/streams/tests/StreamsSmokeTest.java

```diff
@@ -62,16 +65,23 @@ public static void main(final String[] args) throws InterruptedException, IOExce
                 final int numKeys = 10;
                 final int maxRecordsPerKey = 500;
                 if (disableAutoTerminate) {
-                    SmokeTestDriver.generate(kafka, numKeys, maxRecordsPerKey, false);
+                    generate(kafka, numKeys, maxRecordsPerKey, Duration.ZERO, false);
```

Comment:
since this is just going to run until stopped, there's no need (or any sensible value) for delay.